src/components/measure/MeasureService.js

```diff
@@ -239,48 +240,52 @@ goog.require('ga_measure_filter');
         // Register events on map, layer and source to manage correctly the
         // display of overlays
         this.registerOverlaysEvents = function(map, layer) {
+          if (!(layer instanceof ol.layer.Vector)) {
+            return;
+          }
           map.getLayers().on('remove', function(evt) {
             if (evt.element === layer) {
-              var features = evt.element.getSource().getFeatures();
-              for (var i = 0; i < features.length; i++) {
-                this.removeOverlays(features[i]);
-                features[i].set('overlays', undefined);
+              var i, features = evt.element.getSource().getFeatures();
+              while (i = features.pop()) {
```

Comment:
last feature poped is undefined so you actually do `evt.element.getSource().removeFeature(undefined)`
I guess this is ok on ol side, but maybe `if (i){}` would be safer.